analysers/analyser_osmosis_parking_highway.py

```diff
@@ -57,6 +59,40 @@
   highway.id IS NULL
 """
 
+
+sql20 = """
+SELECT
+  parking.id,
+  max(parking_way.id),
```

Comment:
If you wish you can list all access using array_agg with `any_full` or `array_full` in place of `way_full`.

Reply:
I noticed any_full isn't documented, so I made a separate PR for this :)